tslint-server/package.json

```diff
@@ -16,7 +16,7 @@
     "@types/node": "^6.0.41",
     "@types/semver": "^5.3.30",
     "typescript": "^2.0.3",
-    "tslint": "^3.15.1"
+    "tslint": "^4.0.1"
```

Comment:
The server doesn't bundle tslint, but needs to have a development dependency so that it gets the latest version of the API. This change makes sure that server.ts is compiled against the tslint 4.0 API.